static/app/components/modals/inviteMembersModal/index.spec.tsx

```diff
@@ -234,6 +235,66 @@ describe('InviteMembersModal', function () {
     expect(screen.getByRole('button', {name: 'Send invite'})).toBeDisabled();
   });
 
+  it('sends all successful invites with team default', async function () {
+    jest.mocked(useOrganization).mockReturnValue(org);
+    const createMemberMock = MockApiClient.addMockResponse({
+      url: `/organizations/${org.slug}/members/`,
+      method: 'POST',
+    });
+
+    render(<InviteMembersModal {...modalProps} />);
+
+    await waitFor(() => {
+      expect(screen.getByRole('button', {name: 'Add another'})).toBeInTheDocument();
+    });
+
+    // Setup two rows, one email each, the first with a admin role.
+    await userEvent.click(screen.getByRole('button', {name: 'Add another'}));
+
+    const emailInputs = screen.getAllByRole('textbox', {name: 'Email Addresses'});
+    const roleInputs = screen.getAllByRole('textbox', {name: 'Role'});
+
+    await userEvent.type(emailInputs[0], 'test1@test.com');
+    await userEvent.tab();
+    await selectEvent.select(roleInputs[0], 'Admin');
+
+    await userEvent.type(emailInputs[1], 'test2@test.com');
+    await userEvent.tab();
+
+    await userEvent.click(screen.getByRole('button', {name: 'Send invites (2)'}));
+
+    // Verify data sent to the backend
+    expect(createMemberMock).toHaveBeenCalledTimes(2);
```

Comment:
nit: prefer breaking expectations into multiple test cases - that way you can target specific pieces of logic in each test case - the goal being if you change the logic of this code, it should fail exactly 1 test.